apps/central-scan/frontend/src/components/toggle_test_mode_button.tsx

```diff
@@ -47,7 +47,7 @@ export function ToggleTestModeButton({
     <React.Fragment>
       <Button
         onPress={() => setFlowState('confirmation')}
-        disabled={!canUnconfigure && !isTestMode}
+        disabled={!canUnconfigure}
```

Comment:
our `getCanUnconfigure` method on the `store` already contains all this logic, so these extra checks are redundant and may allow us to create behavior in tests with non-existent combinations of parameters